src/main/scala/beam/router/graphhopper/GraphHopperWrapper.scala

```diff
@@ -62,93 +64,112 @@ class GraphHopperWrapper(
     val alternatives = if (response.hasErrors) {
       Seq()
     } else {
-      response.getAll.asScala.map(responsePath => {
-        var linkIds = IndexedSeq.empty[Int]
-        var totalTravelTime = (responsePath.getTime / 1000).toInt
-        val ghLinkIds: IndexedSeq[Int] =
-          responsePath.getPathDetails
-            .asScala(Parameters.Details.EDGE_ID)
+      response.getAll.asScala
+        .map(responsePath => {
```

Comment:
nitpic: the body of map function is pretty big, please, consider to split it to method(s)

Reply:
fixed